### ISSUE TITLE
DocCommentSpacingSniff confused when descripion is not the first in phpDoc

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#483]
The following code is not fixed:
```php
/**
 * @throws FooException
 * Description
 * @throws BarException
 */
function a() : void
{
}
```
Description stays in between of those tags. No newlines are added either.
Repro config:

```xml
    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
        <properties>
            <property name="linesCountBeforeFirstContent" value="0"/>
            <property name="linesCountAfterLastContent" value="0"/>
            <property name="linesCountBetweenDescriptionAndAnnotations" value="1"/>
            <property name="linesCountBetweenAnnotationsGroups" value="1"/>
            <property name="annotationsGroups" type="array">
                <element value="@param"/>
                <element value="@return"/>
                <element value="@throws"/>
            </property>
        </properties>
    </rule>
```

**Comment by kukulich** [comment]
I'm not sure if this is fixable. I think that `Description` is detected as the part of the first `@throws` annotation.
